components/gitpod-protocol/src/public-api-converter.ts

```diff
@@ -542,4 +545,25 @@ export class PublicAPIConverter {
                 return ""; // not allowed
         }
     }
+
+    toSCMToken(t: Token): SCMToken {
```

Comment:
Let's add unit test for it

Reply:
@mustard-mh, sorry server related changes are part of https://github.com/gitpod-io/gitpod/pull/19098
Here it's only about the dashboard. But is still look into this comment in the other PR.
I had a stacked PR, and merged first, now the rest is mixed up. :-(